packages/orbit-components/src/Toast/index.jsx

```diff
@@ -0,0 +1,13 @@
+// @flow
+import { toast } from "react-hot-toast";
+
+import ToastRoot from "./ToastRoot";
+
+import type { createToastType, createToastPromiseType } from ".";
+
+const createToast: createToastType = (content, options) => toast(content, { ...options });
+const createToastPromise: createToastPromiseType = (content, options) =>
+  toast.promise(content, { ...options });
+
+export default ToastRoot;
```

Comment:
If we're switching to all named exports, this needs to be exported as a named export, not default, to reflect what is in the readme.